.circleci/config.yml

```diff
@@ -24,6 +24,15 @@ workflows:
       - header-check
       - linux-benchmarks-basic
 
+  # Store benchmarks to conbench on main
```

Comment:
n00b question, but how does circleCI know when to run this? Is it that this runs by default every time something is merged?

Reply:
Yeah, kind of — it only runs on commits to main so either a merge (or if someone commits directly, that would trigger it too). We'll need to do something slightly different for running on PRs when we get to that, but for now this'll build up our history.

https://circleci.com/docs/configuration-reference#filters-1